libr/crypto/p/crypto_bech32.c

```diff
@@ -0,0 +1,243 @@
+/* Copyright (c) 2017, 2021 Pieter Wuille
```

Comment:
Use one line license like all the other c files in r2, license must be in sync with the metadata in thebplugin struct below